paper-swatch-picker.html

```diff
@@ -204,6 +204,15 @@
         },
 
         /**
+         * The name of the icon to use for the button used as a dropdown trigger.
+         * The name should be of the form: `iconset_name:icon_name`.
```

Comment:
Can you add a note that the user is responsible for importing the iconset correctly, so that the icon actually exists?

Reply:
"Iconset must be imported before use."? Not really sure how to phrase that haha.

Reply:
@notwaldorf Just came across this and realized it's been a while so I added a comment.